app/views/categories/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :title, 'My-Exp | Categories' %>
+<div class="grand-total">
+        <p class="amount-label">Total Amount</p>
+        <p class="amount"><%= "$#{@total}" %></p>
+</div>
+
```

Comment:
Your design is coming up well, and the app functions great! However, we can further improve upon the design aspect by introducing the following tweaks below: (Please take note that it's one of the project's objective that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=))
- [ ] To add a category or a transaction, I had to scroll further down to hit the big green button. Let's ensure that the app perfectly on mobile screens.
- [ ] The red logout button appears rather too bulky to fit on the header. It will be good idea if we can fit the logout button within the  hamburger menu.
- [ ] The hamburger menu and the search icons appears bulky within the header. Let's endeavor to improve upon their size accordingly, like seen in the template design.
- [ ] Let's adjust the font size, background colors and layout to match the provided template design.

**Improve the header design:**
**Green button should stick to the bottom of the page:**
**Your design:**        |  **Your design:**        |  **The original design:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/221381384-68369534-5652-42be-9de8-5aee87dd62ab.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/221381428-ec98106b-a12e-4b21-956b-86f146717e13.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/221381527-6925be5b-0738-406c-b563-763b08337fb2.png">